tests/dogtag/pytest-ansible/requirements.txt

```diff
@@ -6,5 +6,17 @@ pytest-autochecklog==0.2.0
 configparser==4.0.2
 pytest-html==1.22.1
 pytest_marker_bugzilla==0.9.4
-python-bugzilla==2.5.0
-pyOpenSSL
\ No newline at end of file
+pyOpenSSL
+
+# Remove python-bugzilla since it's causing the following error:
+#
+#   ERROR: Command errored out with exit status 1:
+#    command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-mlunmrjl/python-bugzilla/setup.py'"'"'; __file__='"'"'/tmp/pip-install-mlunmrjl/python-bugzilla/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-vn6054r5/install-record.txt --single-version-externally-managed --compile
+#        cwd: /tmp/pip-install-mlunmrjl/python-bugzilla/
+#   Complete output (4 lines):
+#   running install
+#   running build
+#   Generating man/bugzilla.1
+#   error: [Errno 13] Permission denied: '/usr/local/bin/rst2man.py'
```

Comment:
To me this sounds like it isn't executable.

Reply:
This actually can be fixed by adding `python3-docutils`, so I'll revert this change.